control/static/css/base.css

```diff
@@ -0,0 +1,512 @@
+table {
+    border-collapse: separate;
+    border-spacing: 0;
+    display: block;
+    max-width: fit-content;
+    max-height: 80vh;
+    margin: 0 auto;
+    overflow: auto;
+    white-space: nowrap;
+}
+table tr th {
+    position: sticky;
+    top: 0;
+    background-color: var(--color-9);
+}
+table tr th,
+table tr td {
+    border-right: 1px solid var(--gray-8);
+    border-bottom: 1px solid var(--gray-8);
+    padding: 8px 12px;
+}
+table tr th:first-child,
+table tr td:first-child {
+    border-left: 1px solid var(--gray-8);
+}
+table tr th:first-child,
+table tr td:first-child {
+    border-left: 1px solid var(--gray-8);
+}
+table tr th {
+    text-align: left;
+    border: 0.5px solid var(--gray-8);
+}
+table tr:first-child th:first-child {
+    border-top-left-radius: 4px;
+}
+table tr:first-child th:last-child {
+    border-top-right-radius: 4px;
+}
+table tr:last-child td:first-child {
+    border-bottom-left-radius: 4px;
+}
+table tr:last-child td:last-child {
+    border-bottom-right-radius: 4px;
+}
+table thead tr {
+    background-color: var(--color-9);
+    font-weight: 500;
+    text-align: left;
+}
+table tbody tr:nth-child(even) {
+    background-color: var(--color-9);
+}
+table tbody tr:hover {
+    background-color: var(--color-8);
+}
+
+/* Body */
+body {
+    font-family: Roboto, "Microsoft Sans Serif", "Helvetica Neue", Arial, sans-serif;
+    font-weight: normal;
+    color: var(--gray-3);
+    background: var(--gray-9);
+}
+
+
+/* Header */
+header {
+    background-color: var(--color-1);
+    border: 1px solid var(--gray-2);
+    margin: auto;
+    vertical-align: center;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
+header a {
+    color: var(--gray-7);
+}
+header a:hover {
+    color: var(--gray-9);
+}
+header li.logo {
+    color: var(--gray-9);
+    font-size: 1.5em;
+    font-weight: 300;
+    margin-right: 30px;
+}
+header li.logo b {
+    color: var(--highlite-blue);
+    font-weight: 400;
+}
+header li.selected {
+    position: relative;
+}
+header li.selected span.material-symbols-outlined,
+header li.selected a {
+    font-weight: 600;
+    color: var(--gray-9);
+}
+header li.selected a::after {
+    content: "◢◣";
+    color: var(--gray-9); 
+    position: absolute;
+    letter-spacing: -2.5px;
+    bottom: -8px;
+    left: calc(50% - 0.5em);
+}
+header img {
+    vertical-align: middle;
+}
+header nav ul {
+    white-space: nowrap;
+}
+header li {
+    display: inline-block;
+    padding: 10px 10px;
+    border-style: 1px solid var(--color-3);
+    position: relative;
+}
+header a span.material-symbols-outlined {
+    vertical-align: middle;
+    font-size: 1em;
+    padding: 3px;
+    margin-top: -3px;
+}
+
+/* Main */
+main {
+    padding: 50px 0 0;
+    margin: auto;
+    overflow: auto;
+    width: 100%;
+    max-width: fit-content;
+}
+
+main * {
+    margin: 0 min(5vw, 10px);
+}
+
+/* Footer */
+footer {
+    height: 0;
+}
+
+@media only screen and (max-width: 690px) {

```

Comment:
We might want to move away from these soonish and use something more mainstream, thought I'll admit I haven't touched this stuff in a long long time, hence my preference for libs I knew from back in the day that still exist, e.g. boostrap. We can't feasibly test every single browser and version, so we should delegate this responsibility to people who's full time job is to do that kind of stuff, e.g. bootstrap developers.